.semaphore/semaphore.yml

```diff
@@ -114,3 +101,11 @@ blocks:
         - name: Live Gmail tests
           commands:
             - npm run-script test_ci_chrome_consumer_live_gmail
+      epilogue:
+        on_fail:
+          commands:
+            - |
+              if [ -f build/test/test/debugArtifacts/debugHtmlAttachment-0.html ]; then
+                echo "Uploading debug files as job artifacts..."
+                artifact push job build/test/test/debugArtifacts/debugHtmlAttachment-0.html
+              fi
```

Comment:
@rrrooommmaaa Any reason why we didn't store `Live test` artifacts before?

Reply:
None that I know of.

Reply:
I see